### ISSUE TITLE
xpath or accessibility features other than name does not work for some elements in consequence run

### ISSUE THREAD

**Issue by ktufan** [microsoft/WinAppDriver#1902]
Hi,

I have a problem to access elements in a Pytest run from console. the details are below.

I have test cases in two modules and inside two classes. I have fixture to open the app by using driver and the scope of this fixture is "class". When I run the test from PyCharm individually, it find out every elements and run perfectly. But if I want to run both modules. then I go to terminal and run from there. Test cases in the first module (also class) run without issue. since the fixture is set as "class" scope, it tears down and reopen the app. When it open the app again. the driver is acting on some items but I have a list items set, it does not find xpath for that items. Only name features can be used (this is happening only from console (terminal) run, it is not an issue in individual run from PyCharm, as stated above). I cannot use the name property because I am actually testing if it's name is that or different. Moreover, the name property is the value in the list view item that should change in program by setting a new value, generally. xpath or automationid are more stable but they are not recognized for the second module (class also). 

Sorry I have tried to describe the problem as much as I can. 

Please help me with that issue

inoterpreter setting:
python : 3.10
pytest : 7.3.1
selenium: 3.141.0
appium: 1.3.0

**Comment by anunay1** [comment]
I am unbale to understand the issue. can you share you code?
